### ISSUE TITLE
Instance groups model semantics are incorrect

### ISSUE THREAD

**Issue by Temikus** [fog/fog-google#132]
@areznikov-kabam seems to be right in #115 , `instance_groups` methods include:
```
def add_instance(params)
def remove_instance(params)
def get_instances(params)
```
, which should be in `instance_group` model according to current structure.

Since people are relying on those already, I propose either:
- A change in the next major version.
or/and
- Add the correct methods to `instance_group` model too and add deprecation warnings to  `instance_groups`.

@icco @plribeiro3000 - WDYT?

**Comment by plribeiro3000** [comment]
Does the ones at `instance_groups` work at all?

If the don't just remove them and add in the right place otherwise i guess for now we should add the methods on the right place and deprecate the wrong ones.

**Comment by Temikus** [comment]
@plribeiro3000 That's the problem, the ones in `instance_groups` **do** work. So we'll need to deprecate them somehow eventually. I deprecated only the `get_instances` from the collection since it wasn't working anyway.

**Comment by plribeiro3000** [comment]
Hmmm. Lets deprecate them. And add the correct ones.
Then we do a minor release and on a Major one we remove the deprecated apis.

**Comment by Temikus** [comment]
@plribeiro3000 I already did ;) See the #133

**Comment by plribeiro3000** [comment]
👍

**Comment by Temikus** [comment]
Fixed by #133
